cypress-tests/cypress.config.js

```diff
@@ -1,7 +1,17 @@
+const { defineConfig } = require("cypress");
+const fs = require("fs-extra");
+const path = require("path");
+
 let globalState;
-module.exports = {
+// Fetch from environment variable
+const connectorId = process.env.CYPRESS_CONNECTOR;
+const reportName = process.env.REPORT_NAME || `${connectorId}_report`;
```

Comment:
With respect to this change, user can pass custom name to file by exporting the environment variable.  

> https://github.com/juspay/hyperswitch/pull/5563#issuecomment-2277382467

Change requested in this comment can be addressed with this.  

Execute:

```sh
export REPORT_NAME=<connector_name>_report_<HyperswitchVersion>
```

because this is how we're expecting to use this variable.

If the variable is not passed, by default, `<connector_name>_report` will be considered.